packages/slate-editor/src/modules/nodes-hierarchy/quiries/isPlainText.ts

```diff
@@ -0,0 +1,6 @@
+import { Text } from 'slate';
+import type { NodeEntry } from 'slate';
+
+export function isPlainText([node]: NodeEntry) {
```

Comment:
Conceptually `isEmptyTextChild()` and `isPlainText` are checking for the same node type, with some nuances. So their naming should be semantically close.

`isTextNode` and `isEmptyTextNode` sounds like a good choice?